core/src/main/java/com/digitalpebble/stormcrawler/protocol/selenium/RemoteDriverProtocol.java

```diff
@@ -72,13 +72,20 @@ public void configure(Config conf) {
             try {
                 RemoteWebDriver driver =
                         new RemoteWebDriver(new URL(cdaddress), capabilities, tracing);
+                // setting timouts
+                // see https://www.browserstack.com/guide/understanding-selenium-timeouts
                 Timeouts touts = driver.manage().timeouts();
-                int implicitWait = ConfUtils.getInt(conf, "selenium.implicitlyWait", 0);
```

Comment:
Shall we add a (temporary) logging for these old settings for a smooth transision?

Something like

```
selenium.implicitlyWait is deprecated. Please use selenium.timeouts!
```

and fail hard?